src/ducks/transactions/Transactions.jsx

```diff
@@ -283,6 +283,11 @@ class TransactionsD extends React.Component {
   constructor(props) {
     super(props)
     this.scrollSpy = new ScrollSpy(this.getScrollingElement)
+    this.handleScroll = (isIOSApp() ? debounce : throttle)(
+      this.handleScroll.bind(this),
+      300,
+      { leading: false, trailing: true }
+    )
```

Comment:
For my personal culture, what's different between doing this in the constructor VS in a class property ?

Reply:
It's mainly for testing and mocking. It is impossible to mock class via property (as far as I tried) since the method is created is the constructor and not in the prototype.